### PR TITLE
Add Serendipity HQ AWS SES Monitor Bundle.

### PR DESCRIPTION
**Pull request by Aerendir**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by Aerendir** [comment]
This is ready to be merged, if yoou like. Thank you :)

**Comment by Aerendir** [comment]
This PR is ready to be merged.

File: serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Aws Ses Monitor
+shq_aws_ses_monitor:
```
Comment (Nyholm) [inline serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml]: I suggest less "default config" and rather link the user to the bundle's documentation.

**Comment by Nyholm** [comment]
Any updates on this PR?

#eu-fossa

**Comment by Aerendir** [comment]
@Nyholm , excuse me for the extreme late with which I reply but I missed the notification from GitHub about your comment -.-'.

Anyway, I will continue the discussion in your suggestion, so we can try to get a conclusion 💪

**Comment by Nyholm** [comment]
Any updates here?

File: serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml (hunk 1 above)
Comment (Aerendir) [reply serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml]: @Nyholm , what do you suggest? Should have I to completely remove all the default configuration?

**Comment by Aerendir** [comment]
@Nyholm , I replied in the change request you made: https://github.com/symfony/recipes-contrib/pull/531#pullrequestreview-176051655

File: serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml (hunk 1 above)
Comment (Nyholm) [reply serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml]: Yes please

Comment (Aerendir) [reply serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml]: Ok, give me some time... I'll try to fix it by next Tuesday (2019/07/02).

Comment (Nyholm) [reply serendipity_hq/aws-ses-monitor-bundle/2.0/config/packages/shq_aws_ses_monitor.yaml]: No rush. 

Thank you

**Comment by Nyholm** [comment]
Friendly ping =)

**Comment by Aerendir** [comment]
I need some time: in this moment I'm not able to work on this. Please, be patient 🙏

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Nyholm** [comment]
I will close this since it has been open for almost 18 months. 

Feel free to open a new PR when ready.

**Pull request closed without merging**